src/pages/tutorials/getting-started-with-spring-boot.md

```diff
@@ -230,88 +391,15 @@ Overall the process of creating a `GraphQL` and `GraphQLSchema` instance looks l
 
 ![Creating GraphQL](/img/graphql_creation.png)
```

Comment:
do we have this image?

Reply:
Yes the image is at `static/img/graphql_creation.png`. I'm reusing the old diagram